ComfyUI/tests/api_nodes/test_runway_integration.py

```
@@ -0,0 +1,141 @@
+    # Add the current directory to Python path
+    current_dir = os.path.dirname(os.path.dirname(__file__))
+    if current_dir not in sys.path:
+        sys.path.insert(0, current_dir)
+
+def test_runway_node_file_content():
+    """Test that the RunwayText2ImgNode class exists in the file with expected content."""
+    file_path = "comfy_api_nodes/nodes_runway.py"
+    if not os.path.exists(file_path):
+        pytest.skip(f"File {file_path} does not exist")
+    
+    print(f"File {file_path} exists")
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)